apps/testrunner/queue.py

```diff
@@ -0,0 +1,105 @@
+import json
+import time
+
+import requests
+import sys
+
+from .logger import logger
+from .runner import SimpleTest
+
+
+class TaskRepo(object):
```

Comment:
FakeTaskRepo is basically a manual mock for TaskRepo. Normal mocking probably should be used in this case.